src/styles/dropdown.ts

```diff
@@ -17,43 +17,89 @@ export const dropdownCss = css`
     position: absolute;
     z-index: 1000;
 
+    // scroll
+    max-height: 80vh;
+    overflow-y: auto;
+
     hr {
       background-color: ${color.medium};
       border: 0;
       height: 1px;
       margin: 0;
       width: 100%;
     }
-
-    button {
-      align-items: center;
-      background-color: transparent;
-      border-radius: 0;
+  }
+  .tsml-dropdown {
+  
+    &__expand {
+      background-color: ${color.light};
+      padding: 0.25rem 1rem;
       border: 0;
-      color: var(--text);
       cursor: pointer;
+      svg {
+        vertical-align: middle;
+      }
+      &::after {
+        border-bottom: 0;
+        border-left: 0.4em solid transparent;
+        border-right: 0.4em solid transparent;
+        border-top: 0.4em solid ${color.dark};
+        content: '';
+        display: inline-block;
+        vertical-align: 0.2em;      
+      }
+      &[data-expanded="true"]::after {
+        transform: rotate(180deg);
+      }
+      &:hover {
+        background-color: ${color.medium};
+      }
+    }
+
+    &__item {
       display: flex;
-      font-size: var(--font-size);
-      gap: 16px;
-      justify-content: space-between;
-      margin: 0;
-      padding: ${size.gutter / 2}px ${size.gutter}px;
-      text-align: left;
-      white-space: normal;
-      width: 100%;
+      flex-direction: row;
+      flex-wrap: none;
+      border-bottom: 1px solid ${color.light};
 
-      span {
-        background-color: ${color.medium};
-        border-radius: var(--border-radius);
+      .tsml-dropdown__button {
+        flex: auto;
+        align-items: center;
+        background-color: transparent;
+        border-radius: 0;
+        border: 0;
+        //display: flex;
```

Comment:
nbd but to follow conventions if you wouldn't mind sorting these lines (vscode has a "Sort Lines Ascending" option) and removing the commented properties